.github/workflows/tests.yaml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
I would support testing on MacOS before Windows. If it turns out Windows is too complicated to run ScienceWorld on, I don't mind not supporting it.

Reply:
Ultimately, this isn't a problem with ScienceWorld itself (as far as I can tell). ScienceWorld should have absolutely no problems running on Windows. It's just an issue with testing on windows, not the performance of the actual library.

Reply:
Great :)